_data/kathrynparadis/kathrynparadis.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kathryn Paradis
+institution: University of Hawaii at Manoa
+profile_pic: kat.jpg
```

Comment:
I missed this. `jpg` should be `JPG`

Reply:
The change has been made. Please let me know if there's anything else to change.